### ISSUE TITLE
__TEST__ is not defined

### ISSUE THREAD

**Issue by bastiRe** [krakenjs/post-robot#33]
Hi,
first of all thanks for open sourcing this. We were in the process of implementing something similar and this saves us a lot of time.

We're using this via NPM in a project compiled with Webpack and we get the following error: 
```
Uncaught ReferenceError: __TEST__ is not defined
    at Object.__webpack_exports__.b (post-robot.js:1414)
    at __webpack_require__ (post-robot.js:12)
    at Object.__webpack_exports__.i (post-robot.js:37)
    at __webpack_require__ (post-robot.js:12)
    at Object.<anonymous> (post-robot.js:570)
    at __webpack_require__ (post-robot.js:12)
    at Object.<anonymous> (post-robot.js:363)
    at __webpack_require__ (post-robot.js:12)
    at Object.<anonymous> (post-robot.js:435)
    at __webpack_require__ (post-robot.js:12)
```
As soon as we define `__TEST__` everything runs smoothly, is that intended?

**Comment by bluepnume** [comment]
Oh weird, looks like there was some regression here, `webpack.DefinePlugin` should be replacing that with a build-time value. Let me check into this real quick...

**Comment by bluepnume** [comment]
Fixed and published `8.0.4`. Thanks for the heads-up!
